src/components/Header/styled.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { Bars, User, Heart } from '../Icons';
+
+const HeaderContainer = styled.div`
+  height: 60px;
```

Comment:
Using relative units for margins might work best for responsiveness:
https://medium.com/dev-genius/a-bite-sized-best-practices-guide-for-css-units-em-rem-px-and-more-f621c0d56df4